tests/sentry/workflow_engine/endpoints/test_organization_data_condition_index.py

```diff
@@ -0,0 +1,97 @@
+from dataclasses import dataclass
+from unittest.mock import patch
+
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.silo import region_silo_test
+from sentry.utils.registry import Registry
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.types import DataConditionHandler
+
+
+class OrganizationDataConditionAPITestCase(APITestCase):
+    endpoint = "sentry-api-0-organization-data-condition-index"
+
+    def setUp(self):
+        super().setUp()
+        self.login_as(user=self.user)
+        self.registry = Registry[DataConditionHandler](enable_reverse_lookup=False)
+        self.registry_patcher = patch(
+            "sentry.workflow_engine.registry.condition_handler_registry",
+            new=self.registry,
+        )
+        self.registry_patcher.start()
```

Comment:
I mock the registry here, but this only appears to be working when I run the tests locally (`test_group_filter` fails in CI) ?

Reply:
fixed by changing the import location!!

https://stackoverflow.com/questions/65292022/why-does-mock-patch-only-work-when-running-specific-test-and-not-whole-test-suit